kubernetes/windows/Dockerfile

```diff
@@ -15,8 +15,7 @@ ENV chocolateyVersion 1.4.0
 RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 # Fluentd depends on cool.io whose fat gem is only available for Ruby < 2.5, so need to specify --platform ruby when install Ruby > 2.5 and install msys2 to get dev tools
 RUN choco install -y ruby --version 3.1.1.1 --params "'/InstallDir:C:\ruby31'" \

```

Comment:
add "/NoDoc" to --params which will avoid getting docs getting installed.

Reply:
choco does not support noDoc params: https://community.chocolatey.org/packages/ruby